builder/builder.go

```diff
@@ -84,6 +84,16 @@ type Builder struct {
 	UpstreamVerUint32 uint32
 }
 
+// UpdateParameters contains the configuration parameters for building an update
+type UpdateParameters struct {
```

Comment:
Whilst the names are very good, it wouldn't hurt to add a comment for each field.

Reply:
Sounds good. I'll update